app/Commands/VisitCommand.php

```diff
@@ -68,8 +69,14 @@ public function handle()
             return $response->successful() || $response->redirect()
                 ? self::SUCCESS
                 : self::FAILURE;
-        } catch (RenderableException $exception) {
-            $exception->render();
+        } catch (RenderableException|ConnectionException $exception) {
```

Comment:
Could we catch that exception nearer to where it is thrown and convert it to a renderable exception. This way we don't need to make any changes here.

Reply:
Thanks for reviewing for me and this proposal is great!